src/js/components/Carousel/stories/Multiple.js

```diff
@@ -46,13 +46,14 @@ const View2 = () => {
 };
 
 export const Multi = () => (
-  <Grommet>
-    <Carousel>
-      <View0 />
-      <View1 />
-      <View2 />
-    </Carousel>
-  </Grommet>
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={grommet}>
+  <Carousel controls="arrows">
```

Comment:
The sizing is pretty odd for t he middle items. It scrolls in cut-off then jumps in height when the transition is done.